### PR TITLE
Modified jhtest.py and updated jupyterhub-odh image

### PR DESCRIPTION
**Pull request by mroman-redhat**
Closes: #217 

Pull request for the modified jhtest.py with changes and additional features necessary for testing the reworked UI

### CODE REVIEW

**Comment by crobby** [comment]
This needs a rebase to pick up the change that installs google-chrome on the test image.

File: tests/basictests/jupyterhub/jhtest.py [hunk 1]
```diff
@@ -180,40 +193,72 @@ def login(self):
...
-    
     def stop(self):
         self.driver.get(self.url+ "/hub/home")
         stop_elem = self.driver.find_element_by_id("stop")
         stop_elem.click()
 
     def run_notebook(self, notebook, tab):
         path = notebook.split("/")
+        w = WebDriverWait(self.driver, 90)
+        auth_elem = w.until(EC.presence_of_element_located((By.XPATH, '/html/body/div[1]/div/form/input')))
```
Comment (vpavlin) [inline tests/basictests/jupyterhub/jhtest.py:253]: What does this click?

File: tests/basictests/jupyterhub/jhtest.py [hunk 2]
```diff
@@ -274,4 +319,4 @@ def check_exists_by_xpath(self, xpath):
 if __name__ == "__main__":
     jhs = JHStress()
     jhs.run()
-    jhs.quit()
+    jhs.quit()
```
Comment (vpavlin) [inline tests/basictests/jupyterhub/jhtest.py:322]: Missing new line:)

**Review by vpavlin — CHANGES_REQUESTED** [review]
Let's see what CI thinks:)

**Comment by harshad16** [comment]
the way to get rid of the build-check would be with rebase and then changing the commit hash.
`git pull --rebase upstream master` (upstream pointing to opendatahub.io/odh-manifest) 
and then again changing the commit. 

thanks for the patience on this.

**Comment by crobby** [comment]
/test all
gcp cluster flake

File: tests/basictests/jupyterhub/jhtest.py [hunk 3]
```diff
@@ -275,3 +320,4 @@ def check_exists_by_xpath(self, xpath):
     jhs = JHStress()
     jhs.run()
     jhs.quit()
+
```
Comment (crobby) [inline tests/basictests/jupyterhub/jhtest.py:323]: Looks like a newline is needed here

Comment (mroman-redhat) [reply tests/basictests/jupyterhub/jhtest.py:323]: I see a newline there, maybe it took a bit for the diff to change

Comment (vpavlin) [reply tests/basictests/jupyterhub/jhtest.py:323]: There is probably a whitespace on that newline, so it is not really a "newline".

With the tests being so fragile, I'd say let's merge and fix this in next PR

Comment (mroman-redhat) [reply tests/basictests/jupyterhub/jhtest.py:323]: oh I see, okay got it.

**Review by vpavlin — APPROVED** [review]

**Pull request merged**